doc/api/tracing.md

```diff
@@ -80,6 +80,17 @@ string that supports `${rotation}` and `${pid}`:
 node --trace-event-categories v8 --trace-event-file-pattern '${pid}-${rotation}.log' server.js
 ```
 
+To guarantee that the log file is properly generated after signal events like
+`SIGINT`, `SIGTERM`, or `SIGBREAK`, make sure to have the appropriate handlers
+in your code, such as:
+
+```js
+process.on('SIGINT', function onSigint() {
+  console.info('Received SIGINT.');
+  process.exit();
```

Comment:
This changes the exit code to 0 whereas `SIGINT` usually results in 130 on linux and 0xC000013A on windows.
Maybe change to `process.exit(130)` and/or add some comment.

Reply:
Added code and comment. Please review at your convenience.